.github/workflows/setup-python.yml

```
@@ -24,6 +24,9 @@ jobs:
       - name: Run cmn-python-tools setup
         run: just cmn-python-tools
 
+      - name: Run uv setup
+        run: just cmn-uv
+        
       - name: Run aider setup
         run: just cmn-aider
-        
\ No newline at end of file
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)